PlotsBase/ext/GRExt.jl

```diff
@@ -894,7 +894,10 @@ function gr_set_tickfont(sp, letter::Symbol; kw...)
     )
 end
 
-# size of the text with no rotation
+"""
+Size of the text with no rotation.
+Returns `(width, height)`.
+"""
 function gr_text_size(str)
     GR.savestate()
     GR.selntran(0)
```

Comment:
@jheinen can you give what the numbers returned by `GR.inqtext` mean in terms of the size of the displayed area?
Do they have a unit?
Further down I try to add padding for the labels and somehow I need all kinds of correction factors to make it look good, but I am not sure that it scales well when I change the size of the plot.